recipes/foxglove-websocket/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = ("os", "arch", "compiler", "build_type")
+    generators = ("cmake", "cmake_find_package")
```

Comment:
```suggestion
    generators = ("cmake", "cmake_find_package_multi")
```
it generates cmake config file:
https://docs.conan.io/en/latest/integrations/build_system/cmake/cmake_find_package_multi_generator.html